lib/Controllers/DealsController.js

```diff
@@ -888,15 +887,29 @@ class DealsController {
             visible_to: (input.visibleTo !== null) ? input.visibleTo : null,
         };
 
+        input.title = undefined;
+        input.value = undefined;
+        input.currency = undefined;
+        input.userId = undefined;
+        input.personId = undefined;
+        input.orgId = undefined;
+        input.stageId = undefined;
+        input.status = undefined;
+        input.probability = undefined;
+        input.lostReason = undefined;
+        input.visibleTo = undefined;
+
+        const _formWithDynamicFields = { ..._form, ...input};
+
         // remove null values
-        _apiHelper.cleanObject(_form);
+        _apiHelper.cleanObject(_formWithDynamicFields);
```

Comment:
Nice bonus that after this change it is possible to use `under_score` format for input payload attributes.